backend/src/helpers/crypt/encrypt/encrypt.helper.ts

```diff
@@ -1,8 +1,12 @@
-import { hash } from 'bcrypt';
+import { hash, compare } from 'bcrypt';
 import { USER_PASSWORD_SALT_ROUNDS } from '~/common/constants/constants';
 
 const encrypt = (data: string): Promise<string> => {
   return hash(data, USER_PASSWORD_SALT_ROUNDS);
 };
 
-export { encrypt };
+const cryptCompare = (data: string, encrypted: string): Promise<boolean> => {
```

Comment:
```suggestion
const checkIsCryptsEqual = (data: string, encrypted: string): Promise<boolean> => {
```

and also

move `encrypt` to `backend/src/helpers/crypt/encrypt/encrypt.helper.ts`
move `checkIsCryptsEqual` to `backend/src/helpers/crypt/encrypt/check-is-crypts-equal.helper.ts`

and export bother through backend/src/helpers/crypt/encrypt/encrypt.ts